example/passthrough_hp.cc

```diff
@@ -165,7 +165,7 @@ static Fs fs{};
 #define FUSE_BUF_COPY_FLAGS                      \
         (fs.nosplice ?                           \
             FUSE_BUF_NO_SPLICE :                 \
-            static_cast<fuse_buf_copy_flags>(0))
+            static_cast<fuse_buf_copy_flags>(FUSE_BUF_SPLICE_MOVE))
```

Comment:
Hmm, I wonder if you make this more generic, if fuse_buf_copy() should automatically pick it up from CAP features. At least something to think about.